### DISCUSSION TITLE
post/downloadable-blob-links-app-service-auth

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#58] [Announcements]
# post/downloadable-blob-links-app-service-auth

Mark Heath's Development Blog

https://markheath.net/post/downloadable-blob-links-app-service-auth

**Comment by khdc** [comment]
Have you since learned of any easier ways of doing this? Otherwise, I'm going to try this method.

**Reply by markheath** [reply]
This technique worked just fine for me, so didn't need to explore any other options.
